internal/cmd/beta/sqlserverflex/user/create/create.go

```diff
@@ -0,0 +1,195 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/sqlserverflex/client"
+	sqlserverflexUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/sqlserverflex/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/sqlserverflex"
+)
+
+const (
+	instanceIdFlag = "instance-id"
+	usernameFlag   = "username"
+	databaseFlag   = "database"
+	roleFlag       = "role"
+)
+
+var (
+	rolesDefault = []string{"read"}
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	InstanceId string
+	Username   *string
+	Database   *string
+	Roles      *[]string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates an SQLServer Flex user",
+		Long: fmt.Sprintf("%s\n%s\n%s\n%s\n%s",
+			"Creates an SQLServer Flex user login for an instance.",
+			"The password is only visible upon creation and cannot be retrieved later.",
+			"Alternatively, you can reset the password and access the new one by running:",
+			"  $ stackit sqlserverflex user reset-password USER_ID --instance-id INSTANCE_ID",
+			"Please refer to https://docs.stackit.cloud/stackit/en/creating-logins-and-users-in-sqlserver-flex-instances-210862358.html for additional information.",
+		),
+		Example: examples.Build(
+			examples.NewExample(
+				`Create an SQLServer Flex user for instance with ID "xxx" and specify the username, role and database`,
+				"$ stackit sqlserverflex user create --instance-id xxx --username johndoe --role my-role --database my-database"),
```

Comment:
I think that here it would be important to show how a user can provide multiple roles, e.g how the notation works